__mocks__/fileMock.js

```diff
@@ -1 +1 @@
-module.exports = 'test-file-stub';
+module.exports = 'test-file-stub';
```

Comment:
there are still some es lint errors. @giranezafiacre  you can add a new line at the end of each file to remove the errors